graphite_checker.go

```diff
@@ -383,6 +384,10 @@ func (c *Check) DoCheck(w http.ResponseWriter) {
 	s.Code = resp.StatusCode
 	if resp.Header.Get("Content-Type") != "application/json" {
 		s.Message = "Non-JSON response from Graphite (exception?)"
+		buf := make([]byte, 5000)
+		if n, err := io.ReadFull(resp.Body, buf); n == len(buf) || err == io.ErrUnexpectedEOF {
```

Comment:
Just use ioutil.ReadAll and read the whole thing.

Reply:
Ah, the limited number of bytes was a suggestion from Mike. I'm fine doing it this way.

Reply:
We're not too concerned about an excessive response since this doesn't come from an untrusted party. If you want to have a limit, though, a better way would be to wrap the body with http.MaxBytesReader.

Reply:
Oh, I had no idea that existed. Neat!